python_gdeploy/actions/install_gluster.py

```diff
@@ -0,0 +1,46 @@
+import ConfigParser
+
+from python_gdeploy.wrapper import gdeploy_features as gf
+from python_gdeploy.wrapper.gdeploy_wrapper import cook_gdeploy_config
+from python_gdeploy.wrapper.gdeploy_wrapper import invoke_gdeploy
+
+
+GLUSTERFS_PACKAGES = [
+    "glusterfs",
+    "glusterfs-server",
+    "glusterfs-cli",
+    "glusterfs-libs",
+    "glusterfs-client-xlators",
+    "glusterfs-api",
+    "glusterfs-fuse"
+]
+
+
+def get_glusterfs_repo():
+    config = ConfigParser.SafeConfigParser()
+    config.read('/etc/python-gdeploy/python-gdeploy.conf')
```

Comment:
As mentioned above in dev setup user will have to, copy the sample conf file. In rpm based instance, during installation this will be copied.
In any case this conf file will not be of any use if we invoke python-gdeploy from tendrl as we will pass details like repo, rpm names to be installed from tendrl.